### DISCUSSION TITLE
How to how Popup  modal or SLIDE OUT / IN modal (like cart)

### DISCUSSION THREAD

**Discussion by sdomeradzki** [breezefront/community#10] [1. Help]
Hi,

I used Evolution breeze theme. 
I want create a phtml file with popup modal when we click on button.
I see that current AjaxPro extensions use it. We used side Cart from this extensions.

How we can set that we will have few button with content with this effect? Using off course the compatible code with breeze.

Can you help me how look may phtml file for this?

I have for now this: 

````
<div id="myModal" class="modal">

  <!-- Modal content -->
<div class="modal-content">   
    <span class="close">&times;</span>
      <?php echo $this->getLayout() ->createBlock('Magento\Cms\Block\Block') ->setBlockId('MY_STATIC_BLOCK_ID') ->toHtml(); ?>
  </div>
</div>

**Comment by sdomeradzki** [comment]
But maybe you will show me code which used CART sidebar? (with java)
Can you help me?

**Comment by vovayatsyuk** [comment]
You can do it in the same way as in the Luma theme:

```html
<button class="button primary modal-opener">Show popup</button>

<div style="display:none" data-mage-init='{
        "Magento_Ui/js/modal/modal": {
            "trigger": ".modal-opener"
        }
    }'>
    Modal content
</div>
```

**Reply by sdomeradzki** [reply]
And I not needed JS script enter in phtml file?

**Reply by vovayatsyuk** [reply]
`data-mage-init` contains all the params to initialize modal popup. You don't need to write js in this case.
